yeezyCopper.js

```diff
@@ -22,7 +22,7 @@ var config = {
   'expires_month': 'January', // must be full month name to match adidas.com
   'expires_year': '2001', // Year the Credit Card expires
   'security_code': '404' // a 3 or 4 digit CVV code that is on the back of your Credit Card (4 Digits for AMEX on front)
-  'config.shipfast': true //SET TO TRUE IF YOU WANT OVERNIGHT SHIPPING, COSTS $15 EXTRA. SET TO FALSE FOR NORMAL SHIPPING.
+  'shipfast': true //SET TO TRUE IF YOU WANT OVERNIGHT SHIPPING, COSTS $15 EXTRA. SET TO FALSE FOR NORMAL SHIPPING.
```

Comment:
hmmm. getting this error saying that line 25, where i implemented `shipfast` as a variable in that `config` object 
<img width="1280" alt="screen shot 2016-06-15 at 9 56 54 pm" src="https://cloud.githubusercontent.com/assets/5168320/16103186/37bbf788-3344-11e6-92ab-774b600cf532.png">